test/cts/tool/CTSConverter/src/cts_generator.py

```diff
@@ -349,6 +351,19 @@ def DumpJSTest(model, example, js_fd):
             if t.type == "FLOAT16":
                 t.type = "FLOAT32"
 
+    '''
+    # select specifying type models
+    select_specifying_flag = False
+    if model.operations[0].optype == "CONV_2D" or model.operations[0].optype == "DEPTHWISE_CONV_2D":
+        if model.operands[0].type.type == "TENSOR_QUANT8_ASYMM_SIGNED" and \
+           model.operands[1].type.type == "TENSOR_QUANT8_SYMM_PER_CHANNEL":
+            select_specifying_flag = True
+
+    if select_specifying_flag == False:
+        print ("    skip not select types: %s" %example.examplesName, file = sys.stderr)
```

Comment:
@cuiyanx Please add a blank space 
```python
%example.examplesName
```
to 
```python
% example.examplesName
```
and please update the others formatted strings code using '%'.